substra/sdk/client.py

```diff
@@ -376,12 +376,20 @@ def add_traintuple(self, data, exist_ok=False):
 
         If `exist_ok` is true, `AlreadyExists` exceptions will be ignored and the
         existing asset will be returned.
+
+        If `fake_data` is true and the backend is local, the traintuple runs on fake data.
+        If `fake_data` is true and `n_fake_samples` is set, the traintuple
```

Comment:
What do you think about raising an exception if the backend is remote?

Reply:
Indeed, remote backend and fake_data = True should raise an exception.